### PR TITLE
Fixed controller output dir and config path 

### PR DESCRIPTION
**Pull request by 7thcubic**

### CODE REVIEW

**Comment by sergeyklay** [comment]
@rianorie @phalcon Can u review this PR?

File: scripts/Phalcon/Builder/Controller.php [hunk 1]
```diff
@@ -60,9 +60,11 @@ public function __construct($options)
      */
     public function build()
     {
-        $path = '';
-        if (isset($this->_options['directory']) && $this->_options['directory']) {
-            $path = $this->_options['directory'] . '/';
+        $path = '.\\';
+        if (isset($this->_options['directory'])) {
+            if ($this->_options['directory']) {
+                $path = $this->_options['directory'] . '/';
```
Comment (sergeyklay) [inline scripts/Phalcon/Builder/Controller.php]:
```php
// Reduce slashes :)
$path = rtrim($this->_options['directory'], '/\\') . DIRECTORY_SEPARATOR;
```

**Comment by sergeyklay** [comment]
@rianorie @phalcon ping :)

**Comment by andresgutierrez** [comment]
The PR needs to be rebased

**Comment by sergeyklay** [comment]
@7thcubic ?

**Comment by 7thcubic** [comment]
done.. sorry am busy writing a wysiwyg editor for volt.

**Comment by sergeyklay** [comment]
> wysiwyg editor for volt 
:+1: 

So @andresgutierrez can u please review this PR now?

**Comment by 7thcubic** [comment]
wysiwyg editor is hard to make though. I just finished making the barebones of it to support drag and drop and added capability to emulate window sizes. Have not even started adding support for volt code blocks and stuff. but i have to say, angular js is doing a relatively good job. 

Attempting to first make an independant version, before working on the plugin version, which i hope to use phalcon's annotation to scan for each action's available variables and integrate into it. 

Hopefully by then the annotations bug or what not have been fixed, seems like loads of people having problems with it.

**Comment by sergeyklay** [comment]
@7thcubic Can you please rebase the `controller_directory` branch onto `master` branch and pull again? You may need to use `pull -f`

**Comment by 7thcubic** [comment]
345

Closing this, use that instead

**Pull request closed without merging**